README.md

```diff
@@ -1,5 +1,5 @@
 # Backend server for Home Automation
-
+# dit is eigenlijk compleet nutteloos
```

Comment:
Siebe this looks pointless

Reply:
siebe, we agreed to speak english, this is your second strike, one more and you're out!!!!